aws/src/main/java/org/apache/iceberg/aws/glue/GlueTableOperations.java

```diff
@@ -207,6 +216,15 @@ void persistGlueTable(Table glueTable, Map<String, String> parameters, TableMeta
         SET_VERSION_ID.invoke(updateTableRequest, glueTable.versionId());
       }
 
+      if (!ADDITIONAL_LOCATIONS.isNoop()) {
+        ADDITIONAL_LOCATIONS.invoke(updateTableRequest,
```

Comment:
Also, is `additionalLocations` available in the previous SDK version? We might not need reflection in that case.